requirements.base.txt

```diff
@@ -0,0 +1,227 @@
+#
```

Comment:
I'm unsure if this file should be committed to git. I *think* it shouldn't, but would be curious to hear what others think.

For now, I've included it so people can see the output.